frontend/src/components/design-system/typography/BodyText.vue

```diff
@@ -24,6 +24,19 @@ withDefaults(defineProps<Props>(), {
   color: v-bind(color);
 }
 
+*:deep() a {
```

Comment:
Nebylo by lepší to dát globálně? Nebo každý odkaz bude muset být povinně zabalen do BodyText komponenty?

Reply:
Každý odkaz, který má vypadat jako body text, ano. Jinak si stylování můžou řešit další komponenty po svém.